sycl/source/detail/builtins_relational.cpp

```diff
@@ -458,6 +458,9 @@ __SYCL_EXPORT rel_res_t sycl_host_SignBitSet(s::cl_float x) __NOEXC {
 __SYCL_EXPORT rel_res_t sycl_host_SignBitSet(s::cl_double x) __NOEXC {
   return std::signbit(x);
 }
+__SYCL_EXPORT s::cl_long __vSignBitSet(s::cl_double x) __NOEXC {
+  return -static_cast<s::cl_long>(std::signbit(x));
+}
```

Comment:
I was thinking about calling `std::signbit(static_cast<s::cl_double>(x)` at line 470 as an alternative. Either way is fine.